components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -204,12 +204,19 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('tournament_ticker_name', args.tickername)
 	Variables.varDefine('tournament_organizer', CustomLeague:_concatArgs(args, 'organizer'))
 	Variables.varDefine('tournament_sponsors', args.sponsors)
-	Variables.varDefine('date', ReferenceCleaner.clean(args.edate))
-	Variables.varDefine('tournament_date', ReferenceCleaner.clean(args.edate))
-	Variables.varDefine('tournament_sdate', ReferenceCleaner.clean(args.sdate))
-	Variables.varDefine('tournament_edate', ReferenceCleaner.clean(args.edate))
-	Variables.varDefine('sdate', ReferenceCleaner.clean(args.sdate))
-	Variables.varDefine('edate', ReferenceCleaner.clean(args.edate))
+
+	local date = (not String.isEmpty(args.date)) and ReferenceCleaner.clean(args.date)
+					or ReferenceCleaner.clean(args.edate)
+	local startdate = (not String.isEmpty(args.sdate)) and ReferenceCleaner.clean(args.sdate)
+					or ReferenceCleaner.clean(args.date)
+	local enddate = (not String.isEmpty(args.edate)) and ReferenceCleaner.clean(args.edate)
+					or ReferenceCleaner.clean(args.date)
```

Comment:
There's a bunch of duplicate ReferenceCleaner.clean calls here. Could you clean those up?

Reply:
cleaned in c210609